lib/linter.js

```diff
@@ -105,20 +107,39 @@ module.exports = ({ argv }) => {
       for (const match of matches) {
         // Don't mess up if index is 0
         const line = indexToLine(src, (match.index !== undefined) ? match.index : match.start);
-        console.error(`${file}, line ${line.lineNumber}:\n`);
-        console.error(line.text);
-        console.error(' '.repeat(line.columnNumber - 1) + '^');
-        console.error(getMessage({
-          src,
-          line,
-          match
-        }, options.message) + '\n');
-        if (options.url) {
-          console.error(`See: ${options.url}\n`);
-        }
+        results.push({
+          lineNumber: line.lineNumber,
+          // stripIndent just doesn't work here and I don't know why
```

Comment:
Should work, just doesn't, punted them left. Would like to improve this in future if we can. Still think it's more readable than a whole mess of consecutive console calls, which it replaced.